BUILD.md

```diff
@@ -0,0 +1,65 @@
+# Building the project
+
+We use [Gradle build flavors](http://developer.android.com/tools/building/configuring-gradle.html#workBuildVariants) to enable a number of different build variants of MUSER.
+
+For now, We have one  Gradle "platform" flavor dimensions:
+
+* **Google** = Normal Google Play release
```

Comment:
It doesn't look like there is actually a `google` platform flavor in MUSER the way it's defined in OBA, so I wouldn't say that here. Instead, Shuttle defines the variants along a single dimension `all` in `build.gradle` and then filters out some of the combined variants programatically in `build.gradle` for those that don't make sense (e.g., `devRelease`).

For clarity, please list all the build variants that are possible in the app, and then specify which we are using for MUSER. You can see the final possible combined variants using the "Build variants" window in Android Studio:

![image](https://user-images.githubusercontent.com/928045/96181383-e79ad500-0f01-11eb-9056-af60a87ba94d.png)

Looks like:
* `amazonFreeRelease`
* `amazonPaidRelease`
* `devDebug`
* `freeRelease`
* `paidRelease`

IIRC the only ones that matter for MUSER are `devDebug` and `freeRelease`.

Reply:
Thanks for putting this together. I guess I'll mention these variants and document what we are currently using in the BUILD.md file.